GraphOfOrders.Api/Program.cs

```diff
@@ -1,24 +1,26 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using GraphOfOrders.Api.IoC;
+using GraphOfOrders.Repo;
 using GraphOfOrders.Repo.IoC;
 using GraphOfOrders.Service.IoC;
 using GraphOfOrders.Service;
-using Microsoft.AspNetCore.Identity;
-using GraphOfOrders.Repo;
-using System.Security.Claims;
+
 
 var builder = WebApplication.CreateBuilder(args);
-builder.Services.AddAuthentication().AddBearerToken(IdentityConstants.BearerScheme);
-builder.Services.AddAuthorizationBuilder();
-//check this line
-builder.Services.AddDbContext<OrdersContext>();
-builder.Services.AddIdentityCore<MyUser>()
-    .AddEntityFrameworkStores<OrdersContext>()
-    .AddApiEndpoints();
 
 builder.Services.AddAutoMapper(typeof(MappingProfile).Assembly);
 
+// Add DbContext 

```

Comment:
can we add a extension method to this ?
See AddRepoServices example.

Reply:
Done ✅